src/Microsoft.MobileBlazorBindings.ProtectedStorage/MobileBlazorBindingsProtectedStorageExtensions.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using Microsoft.Extensions.DependencyInjection.Extensions;
+using Microsoft.MobileBlazorBindings.ProtectedStorage;
+using System;
+using Xamarin.Forms;
+
+namespace Microsoft.Extensions.DependencyInjection
+{
+    /// <summary>
+    /// Extension methods to add protected storage for all MBB platforms.
+    /// </summary>
+    public static class MobileBlazorBindingsProtectedStorageExtensions
+    {
+        /// <summary>
+        /// Adds protected Storage to the <see cref="IServiceCollection"/>.
+        /// </summary>
+        /// <param name="services">The <see cref="IServiceCollection"/> instance to add the services to.</param>
+        /// <returns>The service collection.</returns>
+        public static IServiceCollection AddProtectedStorage(this IServiceCollection services)
+        {
+            switch (Device.RuntimePlatform)
+            {
+                case Device.iOS:
+                case Device.macOS:
+                case Device.Android:
+                    services.TryAddSingleton<IProtectedStorage, XamarinProtectedStorage>();
+                    break;
+                case Device.WPF:
+                    services.TryAddSingleton<IProtectedStorage, WindowsProtectedStorage>();
```

Comment:
Because there is no implementation for .net core (or WPF): https://github.com/xamarin/Essentials/tree/main/Xamarin.Essentials/SecureStorage

There is only an implementation for UWP.

Reply:
My bad.